clients/example-ui/src/components/app-sidebar.tsx

```
@@ -54,23 +56,62 @@ const data = {
+    }
+
+    if (error) {
+        return <div className="px-4 py-2 text-sm text-red-500">{error}</div>;
+    }
+
+    console.log(chatItems);
+
+    return (
+        <div className="space-y-1">
+            {chatItems.map((chat) => (
+                <Link to={"/chats/$chatId"} params={{ chatId: chat._id }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)